core/controllers/dashboard_test.py

```diff
@@ -463,6 +463,92 @@ def test_no_explorations(self):
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(response['explorations_list'],[])
+
+        # Testing that creator only visit dashboard without any exploration created
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(len(response['explorations_list']),0)
+        
+        # Now viewing the dashboard without creating the exploration
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(response['explorations_list'],[])
+        self.logout()
+
+    def test_create_single_exploration_and_visit_dashboard(self):
+        self.save_new_default_exploration(
+            self.EXP_ID, self.owner_id, title=self.EXP_TITLE)
+        
+        self.login(self.COLLABORATOR_EMAIL)
+        
+        # Testing the quantity of exploration created and it should be 1
+        response = self.get_json(response.DASHBOARD_DATA_URL)
+        self.assertEqual(len(response['explorations_list']),1)
+        
+        # Now viewing the dashboard after creating single exploration
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(len(response['explorations_list']), 1)
+        self.logout()
+
+    def test_create_two_explorations_delete_one_and_visit_dashboard(self):
+        self.save_new_default_exploration(
+            self.EXP_ID_1, self.owner_id_1, title=self.EXP_TITLE_1)
+        
+        self.save_new_default_exploration(
+            self.EXP_ID_2, self.owner_id_1, title=self.EXP_TITLE_2)
+        
+        self.login(self.COLLABORATOR_EMAIL)
+
+        response = self.get_json(response.DASHBOARD_DATA_URL)
+        
+        # Testing the quantity of exploration and it should be 2
+        self.assertEqual(len(response['explorations_list']),2)
+
+        exp_services.delete_exploration(self.owner_id_1, self.EXP_ID_1)
+
+        # Testing whether 1 exploration left after deletion of previous one 
+        response = self.get_json(response.DASHBOARD_DATA_URL)
+        self.assertEqual(len(response['explorations_list']),1)
+        
+        # Now viewing the dashboard after creating two explorations and deleting one of them
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        self.assertEqual(len(response['explorations_list']), 1)
+        self.logout()
+
+    def test_create_multiple_explorations_delete_all_and_visit_dashboard(self):
+        self.save_new_default_exploration(
+            self.EXP_ID_1, self.owner_id_2, title=self.EXP_TITLE_1)
+        
+        self.save_new_default_exploration(
+            self.EXP_ID_2, self.owner_id_2, title=self.EXP_TITLE_2)
+        
+        self.save_new_default_exploration(
+            self.EXP_ID_3, self.owner_id_2, title=self.EXP_TITLE_3)
+        
+
+        self.login(self.COLLABORATOR_EMAIL)
+        
+        response = self.get_json(feconf.DASHBOARD_DATA_URL)
+        # Testing for quantity of explorations to be 3
```

Comment:
Better to shift this line to one place above to maintain uniformity with your coding style.